engine/gamesys/src/gamesys/scripts/script_window.cpp

```diff
@@ -291,22 +291,99 @@ static int GetSize(lua_State* L)
  */
 static int GetMouseLock(lua_State* L)
 {
-    int top = lua_gettop(L);
+    DM_LUA_STACK_CHECK(L, 1);
+
     bool cursor_visible = dmHID::GetCursorVisible(g_Window.m_HidContext);
     // If cursor is visible, it is not locked
     lua_pushboolean(L, !cursor_visible);
 
-    assert(top + 1 == lua_gettop(L));
     return 1;
 }
 
+/*# get the display scale
+ *
+ * This returns the content scale of the current display.
+ *
+ * @name window.get_display_scale
+ * @return scale [type:number] The display scale
```

Comment:
I'm a bit torn about this tbh. We do a max(scale_x, scale_y) in out code, but glfw actually returns both the horizontal and vertical scale. I think it is ok for now. If we ever have a need for it in the future we can return scale, scale_x, scale_y.